utils/build-mlir-aie.sh

```diff
@@ -40,11 +42,12 @@ cmake -GNinja \
     -DCMAKE_CXX_COMPILER=clang++ \
     -DLLVM_DIR=${LLVM_BUILD_DIR}/lib/cmake/llvm \
     -DMLIR_DIR=${LLVM_BUILD_DIR}/lib/cmake/mlir \
-    -DCMAKE_MODULE_PATH=${CMAKEMODULES_DIR}/ \
+    -DCMAKE_MODULE_PATH=${CMAKEMODULES_DIR}/modulesXilinx \
     -DCMAKE_INSTALL_PREFIX="../${INSTALL_DIR}" \
     -DCMAKE_BUILD_TYPE=Release \
     -DLLVM_ENABLE_ASSERTIONS=ON \
     -DAIE_ENABLE_BINDINGS_PYTHON=ON \
+    -DVitisSysroot="" \
```

Comment:
We should change the Building on X86 instructions section, maybe add a section at the beginning, to explain the possible build scenarios based on combinations of using(or not) a sysroot and Vitis. Depending on what exactly you want to build mlir-aie for you don't require Vitis or you don't require a sysroot but there are combinations for which the lit tests will fail.

That should also make it easier for people starting with mlir-aie to know what to expect and which prerequisites they really need.

Reply:
yes indeed, I was waiting for some feedback on this PR before touching the readme

Reply:
the comments in the util scripts could also use an update after this pr